Makefile

```diff
@@ -9,23 +9,21 @@ export PATH  := $(GOBIN):$(PATH)
 
 .PHONY: install-dependencies
 install-dependencies:
-	@rm -Rf bin
-	go install github.com/cosmtrek/air@latest
```

Comment:
I, personally, don't use air, it doesn't seem to be helpful to me to see my changes as they happen so I don't have any opinion on that one